build_node/uploaders/pulp.py

```diff
@@ -251,10 +254,43 @@ def get_artifacts_list(self, artifacts_dir: str) -> List[str]:
         """
         artifacts = []
         for file_ in super().get_artifacts_list(artifacts_dir):
-            if file_.endswith('.rpm'):
+            if file_.endswith('.rpm') and not only_logs:
                 artifacts.append(file_)
             elif file_.endswith('.log'):
                 artifacts.append(file_)
             elif file_.endswith('.cfg'):
                 artifacts.append(file_)
         return artifacts
+
+    def upload(self, artifacts_dir: str,
+               only_logs: bool = False) -> List[Artifact]:
+        """
+
+        Parameters
+        ----------
+        artifacts_dir : str
+            Path to files that need to be uploaded.
+        only_logs : bool
+            Flag if only logs needs to be uploaded
+
+        Returns
+        -------
+        list
+            List of the references to the artifacts inside Pulp
+
+        """
+        artifacts = []
+        errored_uploads = []
+        for artifact in self.get_artifacts_list(
+                artifacts_dir, only_logs=only_logs):
+            try:
+                artifacts.append(self.upload_single_file(artifact))
+            except Exception as e:
+                self._logger.error(f'Cannot upload {artifact}, error: {e}',
```

Comment:
I'm think that using `logger.exception` instead `logger.error` would be better

Reply:
Fixed